services/backend/src/service/global_protected_area.js

```diff
@@ -0,0 +1,30 @@
+const { globalPAAreaTypes, globalPASEKeys } = require('../util/appropriate_keys');
+
+module.exports = (globalProtectedAreaPersistence) => ({
+  /**
+   * Get the area distribution for each protected area category in a given area
+   *  and if applies, in the given strategic ecosystem
+   *
+   * @param {String} areaType area type
+   * @param {String} areaId area id
+   * @param {String} se strategic ecosystem
+   *
+   * @return {Object[]} Values of protected areas categories
+   */
+  getPAAreas: async (areaType, areaId, se = null) => {
+    const areaCol = globalPAAreaTypes(areaType);
+    const seCol = globalPASEKeys(se);
+
+    const areas = await globalProtectedAreaPersistence.findAreaByPA(
+      areaCol,
+      areaId.toString(),
+      seCol,
+    );
+    return areas.map(({ area, ...cats }) => ({
+      area,
+      type: Object.values(cats)
+        .filter((c) => c !== null)
+        .join(' y '),
+    }));
+  },
+});
```

Comment:
> Why not return the total area for protected areas in the selected geofence at this point? I mean, in order to leave the answer as similar to the previous one.
> 
> I understand to get the percentages in the front as we agreed to do in the coverage case because we need the total geofence area, but i think the value for total area for PAs in geofence is better to be included here.

![imagen](https://user-images.githubusercontent.com/19979979/161120772-92c7baf9-5b74-49fe-b85d-f84ae77125e5.png)

Reply:
The point of calculating it in the backend before was to avoid processing the array in the front, but because of the percentages it now has to be processed, so I don't wee why it's worth it